### ISSUE TITLE
ServiceConfigurationError from Skaffold event handler

### ISSUE THREAD

**Issue by fsladkey** [GoogleCloudPlatform/cloud-code-intellij#2858]
## ServiceConfigurationError from Skaffold event handler

Plugin version: 20.10.1-202
IDE type: GoLand
IDE version: 2020.3 Beta

`class java.util.ServiceConfigurationError at ServiceLoader.java : java.util.ServiceLoader#fail:588`

Stack trace:

```
io.grpc.ManagedChannelProvider: io.grpc.netty.shaded.io.grpc.netty.NettyChannelProvider not a subtype
error.stacktrace
java.util.ServiceConfigurationError: io.grpc.ManagedChannelProvider: io.grpc.netty.shaded.io.grpc.netty.NettyChannelProvider not a subtype
	at java.base/java.util.ServiceLoader.fail(ServiceLoader.java:588)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNextService(ServiceLoader.java:1236)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNext(ServiceLoader.java:1264)
	at java.base/java.util.ServiceLoader$2.hasNext(ServiceLoader.java:1299)
	at java.base/java.util.ServiceLoader$3.hasNext(ServiceLoader.java:1384)
	at io.grpc.ServiceProviders.loadAll(ServiceProviders.java:67)
	at io.grpc.ServiceProviders.load(ServiceProviders.java:42)
	at io.grpc.ManagedChannelProvider.<clinit>(ManagedChannelProvider.java:37)
	at io.grpc.ManagedChannelBuilder.forAddress(ManagedChannelBuilder.java:37)
	at com.google.cloud.tools.intellij.kubernetes.skaffold.events.SkaffoldEventHandler.newManagedChannel(SkaffoldEventHandler.kt:319)
	at com.google.cloud.tools.intellij.kubernetes.skaffold.events.SkaffoldEventHandler.listenEvents(SkaffoldEventHandler.kt:75)
	at com.google.cloud.tools.intellij.kubernetes.skaffold.run.SkaffoldCommandLineState$startProcess$1.invokeSuspend(SkaffoldCommandLineState.kt:189)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(Dispatched.kt:241)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:594)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.access$runSafely(CoroutineScheduler.kt:60)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:740)
```

**Comment by fsladkey** [comment]
Kamino cloned this issue to [GoogleCloudPlatform/cloud-code-intellij-internal](https://github.com/GoogleCloudPlatform/cloud-code-intellij-internal/issues/3087)

**Comment by etanshaul** [comment]
this is has been fixed, and will be out in the next release `20.12.1`
